docs/tutorials/index.md

```diff
@@ -31,12 +31,43 @@ Note that it is just a rough classification, and that there are several minor de
 ### Planning simulation
 
 Planning simulation is a simulation type that uses simple dummy data and tests mainly planning/control components.
+It brings the car to the destination while avoiding pedestrians and surrounding cars.
+
+#### How the planning simulation is achieved
+
+- Generates the path to the destination
+- Controls the car along the generated path
+- Detect humans and surrounding cars for the safe operation
+
+In the real-world operation, we need to detect traffic signs and signals to follow the rules, but in this simulation, we only do path generation, path following, and obstacle avoidance.
 
 ### Rosbag replay simulation
 
-Rosbag replay simulation is a simulation type that uses Rosbag data and tests mainly perception modules.
+Rosbag replay simulation is a simulation type that uses Rosbag data and tests mainly perception modules, such as localization.
 Sometimes it is used for endurance tests by repeatedly playing back the data.
 
+#### Localization
+
+Rosbag simulation performs localization, which is the process of figuring out the vehicle pose on the map or on a reference coordinate system.
+In other words, localization is the process of determining the vehcile pose on a local area map (e.g., map of Tokyo area) or on the earth, or on a specific coordinate system (e.g., the tunnel entrance or the initial pose).
```

Comment:
```suggestion
In other words, localization is the process of determining the vehicle pose on a local area map (e.g., map of the Tokyo area) or on the earth, or on a specific coordinate system (e.g., the tunnel entrance or the initial pose).
```

Reply:
Resolved in https://github.com/autowarefoundation/autoware-documentation/pull/111/commits/10d29fcec76ce017f9b2e51b46d9f51c748bd9e1 and https://github.com/autowarefoundation/autoware-documentation/pull/111/commits/cb27e2c1c5377218c15379697b4398e5132d5a54